packages/styled-components/transform/src/lib.rs

```diff
@@ -40,9 +41,13 @@ pub struct Config {
     #[serde(default)]
     pub top_level_import_paths: Vec<JsWord>,
 
+    /// NOTE: This option is enabled by default in
+    /// `babel-plugin-styled-components` but not in this plugin.
```

Comment:
Can we change default?

Reply:
Yes, but technically it would be a breaking change. I don't think there is any code that would actually be broken by this change though.
Do you think we should change this?

Reply:
I think it's practically fine